content/en/account_management/billing/rum.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Real User Monitoring billing FAQ
-kind: faq
+title: Real User Monitoring Billing
+kind: documentation
```

Comment:
Are we sure we want to change this? i have a feeling it was set to `faq` for a reason. @apigirl ?

Reply:
Possibly, but the file should be in the FAQ folder if it's an FAQ.

Reply:
this change is fine